scripts/update-apk.sh

```diff
@@ -43,15 +43,15 @@ if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
 		if [[ $file = "open-event"* ]]; then
 			continue
 		fi
-		cp $file open-event-dev-${file%%}
+		mv $file open-event-dev-${file%%}
 	done
 fi
 
 # Signing Apps
 
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
     echo "Push to master branch detected, signing the app..."
-    cp app-release-unsigned.apk app-release-unaligned.apk
+    cp open-event-master-app-playStore-release-unsigned.apk app-release-unaligned.apk
```

Comment:
@iamareebjamal this is the change you meant right?

Reply:
Yes, and below

Reply:
updated

Reply:
More remaining